app/jobs/create_redis_namespace_job.rb

```diff
@@ -0,0 +1,7 @@
+class CreateRedisNamespaceJob < ActiveJob::Base
+  def perform(account)
+    name = account.cname
```

Comment:
You'll probably want `account.tenant` here (or some derivative thereof.. not sure what naming restrictions we have here). The cname is user-changeable, and the tenant is our internal UUID for the account.